frame/assets/src/functions.rs

```diff
@@ -660,7 +660,6 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 				accounts: 0,
 				sufficients: 0,
 				approvals: 0,
-				is_frozen: false,
```

Comment:
So this should rather merge into the other MR, since otherwise there would be two version 1 migrations.

Reply:
Yeah, exactly.  This is based off the other PR. But i wanted it to be separate so it can be reviewed easily before I merged it. Otherwise there would just be too much going on in theo other PR